packages/edit-site/src/components/global-styles/test/shadow-utils.spec.js

```diff
@@ -0,0 +1,254 @@
+/**
+ * Internal dependencies
+ */
+import { getShadowParts, shadowStringToObject } from '../shadow-utils';
+
+const colorFormats = {
+	named: 'red',
+	hex: '#ff0000',
+	hexAlpha: '#ff000077',
+	rgb: 'rgb(255, 0, 0)',
+	rgbSpaced: 'rgb(255 0 0)',
+	rgbMultiSpaced: 'rgb(255  0   0)',
+	rgba: 'rgba(255, 0, 0, 0.5)',
+	rgbp: 'rgb(255 0 0 / 50%)',
+	hsl: 'hsl(0, 100%, 50%)',
+	hsla: 'hsla(0, 100%, 50%, 0.5)',
+	hslp: 'hsl(0 100% 50% / 50%)',
+	currentcolor: 'currentcolor',
+
+	// invalidColor1: 'invalid',
+	// invalidColor2: 'hwb(0, 0%, 0%)',
```

Comment:
I noticed these comments have been left in and are not mentioned elsewhere - should they be removed?

Reply:
These are unnecessary in the present context. removed them.